.github/workflows/build-fleetd-tables.yml

```diff
@@ -0,0 +1,76 @@
+name: Build fleetd_tables
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Set up Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          go-version: ${{ vars.GO_VERSION }}
+      - name: Build Linux and Windows binaries
+        run: |
+          GOOS=linux go build -o fleetd_tables_linux.ext ./orbit/cmd/fleetd_tables
+          GOOS=windows go build -o fleetd_tables_windows.exe ./orbit/cmd/fleetd_tables 
+
+      - name: Upload binaries to workspace
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v2
+        with:
+          name: binaries
+          path: |
+            fleetd_tables_linux.ext
+            fleetd_tables_windows.exe
+
+  build-macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Set up Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          go-version: ${{ vars.GO_VERSION }}
+      - name: Build macOS binaries
+        run: |
+          CGO_ENABLED=1 GOOS=darwin GOARCH=amd64 go build -o fleetd_tables_amd64 ./orbit/cmd/fleetd_tables
+          CGO_ENABLED=1 GOOS=darwin GOARCH=arm64 go build -o fleetd_tables_arm64 ./orbit/cmd/fleetd_tables
+      - name: Create universal binary for macOS
+        run: |
+          lipo -create fleetd_tables_amd64 fleetd_tables_arm64 -output fleetd_tables_darwin_universal.ext
+      - name: CodeSign
+        env:
+          CODESIGN_IDENTITY: 51049B247B25B3119FAE7E9C0CC4375A43E47237
```

Comment:
I'm not sure which cert/identity we are using here, but do we foresee issues with customers using it?  ie. If it's the same cert we use to sign the agent and we need to revoke the certificate for some reason, it will affect this extension as well.

Reply:
Good call. We can easily remove this step if it's more trouble than it's worth.